templates/common/infra/bicep/core/monitor/applicationinsights-apim-dashboard.bicep

```diff
@@ -0,0 +1,1352 @@
+param name string
```

Comment:
What makes this one significantly different than the default to warrant a new file?  What do we have to do to make them one file?

Reply:
This is optional. To have a dashboard that includes APIM metrics. I could research to see if we could have just one file and pass the useAPIM param?

Reply:
Can we include those in default and not have to use a param?  It just ignores those section if not valid...

Reply:
Should we remove the specific dashboard for APIM for now?

Reply:
fixed